common/src/main/java/net/william278/huskhomes/config/Settings.java

```diff
@@ -215,6 +215,10 @@ public Optional<String> getSoundEffect(@NotNull SoundEffectAction action) {
     @YamlKey("economy.enabled")
     public boolean economy = false;
 
+    @YamlComment("Use this currency for payments (works only with RedisEconomy), defaults to Vault currency")
+    @YamlKey("economy.currency_name")
+    public String redis_economy_name = "vault";
```

Comment:
No, I meant the _config key_, the variable name should be redisEconomyName, lol; the config key `economy.redis_economy_name`.

Reply:
ah lol ok. done

Reply:
> ah lol ok. done

I don't think you have ;-) The variable name needs to be redisEconomyName

Reply:
> ah lol ok. done

I don't think you have ;-) Variable name should be redisEconomyName?

Reply:
thumb up